contributor/outline.md

```diff
@@ -0,0 +1,46 @@
+# InnerSource Contributor Content Outline
+
+## Introduction/Rationale
+   - Quick Overview of Roles
+      - Explain how contributor role relates to other roles
+   - Discuss the value of contribution
```

Comment:
Seems like we need to also consider the value it brings to the person contributing as well.  I'd like to propose we add the overtly obvious to the list.  

- Directly increasing the impact for the contributor(individual).
- Through the contribution and review process, a contributor can and will level up their own skills, under the mentorship of the Trusted Committers.

Reply:
I concur with this.